packages/strapi-plugin-content-type-builder/admin/src/containers/FormModal/utils/attributes/constants.js

```diff
@@ -0,0 +1,19 @@
+import getTrad from '../../../../utils/getTrad';
+
+const nameField = {
+  autoFocus: true,
+  name: 'name',
+  type: 'text',
+  label: {
+    id: getTrad('modalForm.attribute.form.base.name'),
+  },
+  description: {
+    id: getTrad('modalForm.attribute.form.base.name.description'),
+  },
+  validations: {
+    required: true,
+  },
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export { nameField };
```

Comment:
hmmm, it seems that this file doesn't export some constants. Should we keep the `constant` filename?